include/broker/zeek.hh

```diff
@@ -1,11 +1,25 @@
 #pragma once
 
+#include <cstddef>
+#include <iterator>
+#include <optional>
+
 #include "broker/data.hh"
+#include "broker/detail/assert.hh"
 
 namespace broker::zeek {
 
 const count ProtocolVersion = 1;
 
+/// Metadata attached to Zeek events.
+enum class Metadata : uint8_t {
+  NETWORK_TIMESTAMP = 1,
+  // 1 - 199 is reserved for Zeek. Otherwise free for external
+  // users to experiment with before potentially including
+  // in the reserved range.
+  USER_METADATA_START = 200,
+};
```

Comment:
Two remarks:

1) To me, it's confusing that `Metadata` is a class in Python but an enum in C++ world. It's also not describing the actual meta data but the type thereof. So maybe `MetaDataType` or `MetaDataField`?
2) The enum values are all-uppercase, which should be reserved for macro names. This header is using Zeek's coding style. However, even there, never code seems to use CamelCase for enum identifiers.

Reply:
I've renamed to MetaDataType and camel-cased.

I made the renamed the Python side but kept the class. No strong feelings and having the ready made `broker.Count` instances is convenient.